fabric-tags-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/FabricItemTags.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.item.Item;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.tag.TagRegistry;
+
+/**
+ * The place where the new item tags added by Fabric live.
+ */
+public final class FabricItemTags {
+	/**
+	 * Allows for the addition of more types of Elytra via the tag system.
+	 *
+	 * <p>The textures for the Elytra feature renderer are located at <code>assets/IDENTIFIER_NAMESPACE/textures/entity/elytra/IDENTIFIER_PATH.png</code>
+	 */
+	public static final Tag<Item> ELYTRA = TagRegistry.item(new Identifier("fabric-tags-v1", "elytra"));
```

Comment:
As mentioned in discord, the unbreaking enchantment works differently on armor from all other tools, and there being no way to check this may cause some confusion from developers when they get reports of elytra damaging too quickly. Other than that, a pr to make elytras official is great because of all the redirects.